README.md

```diff
@@ -31,40 +104,55 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
 
-## Learn More
+## 👥 Authors <a name="authors"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+👤 **OMAIDU OTHNIEL**
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- GitHub: (https://github.com/othniel2471)
+- Twitter: (https://twitter.com/@Oomaidu)
+- LinkedIn: (https://www.linkedin.com/in/othniel-omaidu-3350a9164)
 
-### Code Splitting
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+## 🔭 Future Features <a name="future-features"></a>
 
-### Analyzing the Bundle Size
+- [ ] Adding hero image for homepage
+- [ ] Adding dark mode
+- [ ] Use and state management of Asynchronous code
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Making a Progressive Web App
+## 🤝 Contributing <a name="contributing"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Contributions, issues, and feature requests are welcome!
 
-### Advanced Configuration
+Feel free to check the [issues page](https://github.com/Othniel2471/country-details/issues). Open a New Issue and I'll look intently into your suggestions.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Deployment
+## ⭐️ Show your support <a name="support"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+If you like this project **follow me on github** for more amazing projects
+and ideas
 
-### `npm run build` fails to minify
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank **microverse** for the definition of the project requirements and provision of this template
+
```

Comment:
- [ ] Great job adding an acknowledgement to the appropriate section 👏! However, I suggest giving credit to the original designer, [Nelson Sakwa](https://www.behance.net/sakwadesignstudio), who made the design available for free. This not only meets the project requirements but also shows appreciation for their work. They deserve recognition for making their work accessible to us. :ok_hand:  😊